lib/parser/objects.ts

```diff
@@ -3,13 +3,10 @@ import { OLexerToken } from '../lexer';
 import * as I from './interfaces';
 export class OI implements Position {
   protected i_?: number;
-  constructor(parent: ObjectBase | OFile, i: number)
-  constructor(parent: ObjectBase | OFile, line: number, character: number)
-  constructor(parent: ObjectBase | OFile, line: number, character: number, i: number)
   constructor(public parent: ObjectBase | OFile, i: number, j?: number, k?: number) {
     if (j === Number.POSITIVE_INFINITY) {
       const lines = (this.parent instanceof OFile ? this.parent : this.parent.rootFile).lines;
-      j = lines[i].length - 1;
+      j = lines[i]!.length - 1;
```

Comment:
`lines[i]` might actually be undefined, should be checked

Reply:
Then it throws an error, what else do you want to do?